mesonbuild/interpreter.py

```diff
@@ -2131,6 +2131,8 @@ def _find_cached_dep(self, name, kwargs):
         # Check if we've already searched for and found this dep
         if identifier in self.coredata.deps:
             cached_dep = self.coredata.deps[identifier]
+            mlog.log('Cached dependency', mlog.bold(name),
+                    'found:', mlog.green('YES'))
```

Comment:
__[Flake8]__

continuation line under-indented for visual indent


<!-- Comment by [SideCI](https://sideci.com) -->